docker/README.md

```diff
@@ -39,32 +38,44 @@ If your REST Proxy uses self-signed SSL certificates, you can use the
 `PROXY_SKIP_VERIFY=true` environment variable to instruct the proxy to
 not verify the backend TLS certificate.
 
+### Specifying multiple Kafka REST proxies
+
+You can specifying multiple Kafka proxies by separating proxy url's with a space. Consider the following example:
+
+    docker run --rm -it -p 8000:8000 \
+               -e "KAFKA_REST_PROXY_URL=http://kafka.rest.proxy.url http://other-kafka.rest.proxy.url" \
```

Comment:
Isn't comma-separated easier to read, than space separated?

Reply:
I thought of that too. I am thinking though that there is no way a rest proxy url could contain a space but I am not 100% sure if it can contain a comma.

We can use a regex to split the url string and the separator could be space _and_ comma.

Here is the link to the URI RFC. https://www.ietf.org/rfc/rfc2396.txt

What do you think @jglambed ?

Reply:
Whitespace works better since comma can technically exist in URL though it's very rare. Either works for me.